### ISSUE TITLE
Runbot life is too short

### ISSUE THREAD

**Issue by Yajo** [OCA/maintainer-tools#408]
Not sure what's happening lately, but since some weeks ago, the runbot builds last live for about 1-2 hours.

I ignore if some policy has changed, but it was very useful to have them for some days. Sometimes you just go and hit the rebuild button but keep on doing other work until you go back to review, and if you took more than 2 hours, you are out of luck.

**Comment by pedrobaeza** [comment]
No, the policy is the same AFAIK, but too many builds are done, and the recycling happens before.

**Comment by Yajo** [comment]
Are you sure that's the only reason? I've been hitting this *always* since some weeks ago.

**Comment by pedrobaeza** [comment]
Let's see if @gurneyalex has other explanation
